app/js/controller/list_controller.js

```diff
@@ -73,6 +73,9 @@ export default class ListController extends Controller {
       for (let manifestURL in this.list) {
         this.list[manifestURL].installed = !!installedApps[manifestURL];
         this.list[manifestURL].mozApp = installedApps[manifestURL] || false;
+        if (this.detailsView.isShowing(manifestURL)) {
```

Comment:
You might want to comment what this is doing. It looks like a noop in isolation.